web/src/flow/stages/prompt/PromptStage.ts

```diff
@@ -43,13 +43,31 @@ export class PromptStage extends BaseStage<PromptChallenge, PromptChallengeRespo
                     class="pf-c-form-control"
                     ?required=${prompt.required}
                     value="${placeholderAsValue ? prompt.placeholder : ""}">`;
+            case PromptTypeEnum.TextArea:
+                return `<textarea
+                    type="text"
+                    name="${prompt.fieldKey}"
+                    placeholder="${prompt.placeholder}"
+                    autocomplete="off"
+                    class="pf-c-form-control"
+                    style="min-height: 4em; max-height: 15em; resize: vertical;"
```

Comment:
these styles should be set CSS instead of inline, also a very minor thing

Reply:
I am not sure how to do this, frontend has never been my strong suit. Can you please show me?

Reply:
Yeah of course, see https://github.com/goauthentik/authentik/blob/63e3f6545bd0d3cf036e56d93d30add9f57ceaa0/web/src/flow/FormStatic.ts#L22

Add the import `import { css } from "lit";`

and then in the styles at the top use the
```
css``
```
template tag to add css for the `textarea {}` selector